src/mapping/config/MappingConfiguration.cpp

```diff
@@ -204,6 +214,10 @@ MappingConfiguration::MappingConfiguration(
 
   auto attrSolverRtol = makeXMLAttribute(ATTR_SOLVER_RTOL, 1e-9)
                             .setDocumentation("Solver relative tolerance for convergence");
+  // TODO: Discuss whether we wanto to introduce this attribute
+  // auto attrMaxIterations = makeXMLAttribute(ATTR_MAX_ITERATIONS, 1e6)
+  //                              .setDocumentation("Maximum number of iterations of the solver");
```

Comment:
TODO
Do we really need this? I guess a good robust value could be enough. What did we use so far for PETSc, 10,000, right?

Reply:
Yes, in PETSc we use a hard-coded value of 10.000. I'm not sure about a robust default. In principle, the max iteration should depend on the system size . For small systems, reaching this iteration count is a waste of time and for large systems it might be too less.
Do we really need this? I don't have a strong opinion on this. The solver tolerance would be the more natural parameter to modify, I guess.